README.md

```diff
@@ -1,4 +1,4 @@
-![image](images/confluent-logo-300-2.png)
+![image](https://raw.githubusercontent.com/confluentinc/examples/36bae15c8690c6fc008dc5d9a2656379002b48fe/images/confluent-logo-300-2.png)
```

Comment:
@javabrett how about pointing to `latest` instead of `36bae15c8690c6fc008dc5d9a2656379002b48fe`?

i.e.,

https://raw.githubusercontent.com/confluentinc/examples/latest/images/confluent-logo-300-2.png

Reply:
Could do this if best - my understanding is that `lastest` would return `404` if the file is deleted (but from which branch - the default?) - so if we start using that name in URLs, we should ensure that this file is "the one" - curated maybe with a README that explains that it can't be safely deleted as it is relied on externally to the project.

So a trade-off between a permalink to a fixed file vs a maintained file but must be sure not to delete.  Happy to change this.

Reply:
I keep `latest` updated.  If the concern is the dependency, you could remove the dependency completely and just pull the PNG into this repo.

Reply:
Done!